master/static/migrations/20210203150406_change-limit-behavior.up.sql

```diff
@@ -0,0 +1,24 @@
+CREATE OR REPLACE FUNCTION public.page_info(total bigint, "offset" int, "limit" int)
+    RETURNS json
+    LANGUAGE sql IMMUTABLE
+AS $$
+WITH start_index AS (
+    SELECT (CASE WHEN "offset" < 0
+                 THEN total + "offset"
+                 ELSE "offset" END) AS start_index
+), end_index AS (
+    SELECT (CASE WHEN (SELECT start_index FROM start_index) + "limit" > total THEN total
+                 WHEN "limit" = 0 THEN least(100::bigint, total - (SELECT start_index FROM start_index))
```

Comment:
shouldn't this be:
```
WHEN "limit" = 0 THEN least(100::bigint + (SELECT start_index FROM start_index), total)
```
?

Reply:
yeah sorry. i had fixed that and another issue why my pre push hook failed so it didn't push.. stupid gocritic linter.